instructor/function_calls.py

```diff
@@ -17,11 +16,7 @@
 from instructor.exceptions import IncompleteOutputException
 from instructor.mode import Mode
 from instructor.utils import classproperty, extract_json_from_codeblock
-from instructor.validators import (
```

Comment:
I didn't delete validators.py for now because I'd like to reintroduce this in the form of a mixin in a subsequent version of instructor. Just deleting the logic for now in OpenAI Schema and the tests that are written for the OpenAI Schema in a subsequent PR.